conans/test/functional/generators/virtualenv_test.py

```diff
@@ -110,9 +111,7 @@ class PowerShellCommands(object):
 
     @property
     def skip(self):
-        # Change to this once support for PowreShell Core is in place.
-        # skip = not (os_info.is_windows or which("pwsh"))
-        return (not os_info.is_windows) or os_info.is_posix
+        return not (os_info.is_windows or shutil.which("pwsh"))
```

Comment:
This might be Python3 only (shutil.which). Even if we are deprecating Py2, we still run CI with it, just to know. If the Conan ``which()`` works fine, please use it instead. Thanks!

Reply:
Ah yes, fixed!